sequence-classification/jupiter_shared/helpers/datasets_helper.py

```diff
@@ -0,0 +1,60 @@
+import json
```

Comment:
Nie dodawałbym do nazwy pliku `helper`, bo to nie są funkcje, które dostarczają dodatkową funkcjonalność podczas przetwarzania datasetów, one dostarczają te datasety.